package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "powershell-preview",
-  "displayName": "PowerShell Preview",
+  "name": "powershell-test",
+  "displayName": "PowerShell",
   "version": "2021.2.3",
-  "preview": true,
+  "preview": false,
   "publisher": "ms-vscode",
-  "description": "(Preview) Develop PowerShell scripts in Visual Studio Code!",
+  "description": "Develop PowerShell scripts in Visual Studio Code!",
```

Comment:
It makes total sense to me to use a third name for dev builds of the extension.

- Possible alternative to `powershell-test`: `powershell-dev`
- My suggestion would be for `"preview": true`, since anything that isn't stable is essentially a preview of some kind
- Are we still changing the description based on the channel of the extension? If so, might be worth having a note or tag for the dev build
- We probably want something in place to prevent publication of the dev build to the marketplace

Reply:
This was just a test commit, I'm rolling it back lol.